code/harness/Tester.cpp

```diff
@@ -450,6 +450,10 @@ int Tester::test_setup() {
   return test_loader.get_instance()->setup();
 }
 
+int Tester::test_pass(string mount_dir, string filesys_size){
```

Comment:
could you rename this to something else like `init_values` or something? `test_pass` almost makes it seem like you're saying the test passed.